graylog2-web-interface/src/components/event-notifications/event-notification-types/LegacyNotificationForm.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, ControlLabel, FormGroup, HelpBlock } from 'react-bootstrap';
+import lodash from 'lodash';
+
+import { Select } from 'components/common';
+import { ConfigurationFormField } from 'components/configurationforms';
+
+class LegacyNotificationForm extends React.Component {
+  static propTypes = {
+    config: PropTypes.object.isRequired,
+    onChange: PropTypes.func.isRequired,
+    legacyTypes: PropTypes.object.isRequired,
+  };
+
+  propagateMultiChange = (newValues) => {
+    const { config, onChange } = this.props;
+    const nextConfig = lodash.cloneDeep(config);
```

Comment:
I think the following code can be simplified with:

```
const nextConfig = Object.assign({}, config, newValues);
```

That will merge both objects into a new one.

Reply:
:heavy_check_mark: